install/set_gigabyte_node_bmc_to_factory_defaults.md

```diff
@@ -17,28 +19,26 @@ Use the management scripts and text files to reset Gigabyte BMC to factory defau
 
 #### Apply the BMC Factory Command
 
-1. Create a `node.txt` file and add the target node information as shown:
+1.  Create a `node.txt` file and add the target node information as shown:
 
-   Example `node.txt` file with two nodes:
+    Example `node.txt` file with two nodes:
 
-   ```screen
+    ```screen
     10.254.1.11 x3000c0s9b0 ncn-w002
     10.254.1.21 x3000c0s27b0 uan01
-   ```
+    ```
 
-   Example `node.txt` file with one node:
+    Example `node.txt` file with one node:
 
-   ```screen
+    ```screen
     10.254.1.11 x3000c0s9b0 ncn-w002
-   ```
-
-2. Use Redfish to reset the BMC to factory default.
-
-   The BMC is running 12.84.01 or a later version.
+    ```
 
-   ```bash
-   ncn-w001# sh do_Redfish_BMC_Factory.sh
-   ```
+2. Use Redfish to reset the BMC to factory default, the BMC is running 12.84.01 or later version, run:
```

Comment:
Should this say, "if the BMC is running..."?

Reply:
I have the same question. One way or the other, this should be transformed into a proper sentence with a clear meaning, before we merge.

Reply:
Fixed